client-templates/jira/.env.sample

```diff
@@ -7,13 +7,18 @@ JIRA_USERNAME=
 # your personal password to your Jira Server account
 JIRA_PASSWORD=
 
+# your Jira Server hostname, i.e. jira.yourdomain.com
+JIRA_HOSTNAME=
```

Comment:
breaking change for current jira broker users!

Reply:
This won't work in cases where the username has an `@` in it (e.g. an email address). No amount of encoding will help either. This needs to become similar to how we handle the `auth` object as defined in the rules:

```
https://github.com/snyk/broker/blob/master/client-templates/jira/accept.json.sample#L22
```

This is so we can use request's auth handler:
https://github.com/snyk/broker/blob/c0418a3b388e33aa3f2458d3275ae05d1633ce4b/lib/filters/index.js#L137

Reply:
WOW

I didn't think of this at all!

I did however test this for real with our own jira creds, which included an email address as the username - and it worked! So I'd like to keep it this way, or maybe defer jira validation logic to a later time.